src/scripts/changelog.zig

```diff
@@ -17,8 +17,13 @@ pub fn main(shell: *Shell, gpa: std.mem.Allocator) !void {
     const merges = try shell.exec_stdout(
         \\git log --merges --first-parent origin/release..origin/main
     , .{});
-    try shell.project_root.writeFile("./zig-cache/merges.txt", merges);
-    log.info("merged PRs: ./zig-cache/merges.txt", .{});
+
+    if (try shell.dir_exists("./zig-cache")) {
+        try shell.project_root.writeFile("./zig-cache/merges.txt", merges);
+        log.info("merged PRs: ./zig-cache/merges.txt", .{});
+    } else {
+        log.warn("merged PRs: ./zig-cache does not exist; not writing merges.txt", .{});
+    }
```

Comment:
Took a closer look at what we have and recalled why I am conflicted about having `shell.justWriteTheFilePlease` helper! The "capability oriented" `std` API, `dir.writeFile(text)` is just nicely looking syntactically, with `dir` as a method receiver. 

So I think we shouldn't wrap it just yet, so let's change the code to

```suggestion
    try shell.project_root.makePath("./zig-cache");
    try shell.project_root.writeFile("./zig-cache/merges.txt", merges);
    log.info("merged PRs: ./zig-cache/merges.txt", .{});
```

I think the semantics of just creating a directory is the right one, and I think it's a fair game that we re-use `zig-cache` directory here. Semantically, it is conventient to use project-local temporary directory, and, while we can create our own, I think just re-using existing one is simpler.